app/pages/ResourceGuides/ResourceGuides.module.scss

```diff
@@ -0,0 +1,20 @@
+.resourceWrapper {
+  margin: 0 auto;
+  padding: 26px;
```

Comment:
Do you think you could just add a `max-width: 1160px` here? I think that'll be enough to make the page not look weird on a wide browser window.